tests/e2e/models/mnist_train_test/datasets.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 Google LLC
```

Comment:
This doesn't look like the IREE license header. Where is this code coming from?

Reply:
It comes from Jax.

Reply:
I can put the IREE license above it.

Reply:
That's under Google's CLA. We (as in, Google) can relicense it under the IREE license (that's basically why the CLA exists). I'm not sure whether that means that I have to technically be the one to do it though. Let me check with the lawyers.